Content.Server/GameObjects/Components/Mobs/Speech/SpanishAccentComponent.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.CodeAnalysis.Operations;
+using Microsoft.EntityFrameworkCore.Internal;
+using Robust.Shared.GameObjects;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Content.Server.GameObjects.Components.Mobs.Speech
+{
+    [RegisterComponent]
+    public class SpanishAccentComponent : Component, IAccentComponent
+    {
+        public override string Name => "SpanishAccent";
+
+        public string Accentuate(string message)
```

Comment:
This needs to also put an `e` before every or some `s`'s. For example, saying `spanish` with this accent would be turned into `espanish`.

Reply:
`message.Replace(" s", " es");`